tests/test_set_config.py

```diff
@@ -241,10 +241,10 @@ def test_set_config_dry_run(): #pylint: disable=unused-variable
 
     outstring = io.StringIO()
     with redirect_stdout(outstring):
-        set_config.create_config_obj(args)
+        set_config.create_config_file(args)
     result = outstring.getvalue()
 
-    assert result.rstrip('\n') == expected_content.rstrip('\n')
+    assert expected_content.rstrip('\n') in result.rstrip('\n')
```

Comment:
The adoption of the same logging as was in templater.py meant that none of the strings matched exactly anymore. This is just making sure that the expected string is at least part of the output.